### ISSUE TITLE
Affected and independent fog options 

### ISSUE THREAD

**Issue by envysrcs** [momo5502/boiii#531]
### Duplicate Issue Check

- [ ] I have searched the list of issues on this repository. I confirm that this is not a duplicate of any open or closed issue(s).

### Your feature request or suggestion:

These would be a feature I would like to see and I think many others

**Comment by Ahhalrite** [comment]
Agreed. I'm not sure how difficult it is to implement, or if it's asking a lot, but would absolutely change the experience for me.
